x-pack/plugins/lens/public/indexpattern_datasource/loader.ts

```diff
@@ -100,6 +100,7 @@ export function convertDataViewIntoLensIndexPattern(dataView: DataView): IndexPa
       Object.fromEntries(
         Object.entries(fieldFormatMap).map(([id, format]) => [
           id,
+          // @ts-expect-error FIXME Property 'toJSON' does not exist on type 'SerializedFieldFormat'
```

Comment:
@dej611 I remember we were struggling a bit with this - do you know where this is coming from? How can there be a `toJSON` on the format that should be serialized? Was this done to work around some bug?